### ISSUE TITLE
Test Data empty Annotations

### ISSUE THREAD

**Issue by Davo00** [rossumai/docile#78]
Hello @mskl, @simsa-st 

If I understand correctly, the "test" keyword data should be used to evaluate the model. The annotations are present in the training and validation sets, but for test data the annotations look like this: `{
  "metadata": {
    "page_count": 1,
    "page_sizes_at_200dpi": [
      [
        1704,
        2203
      ]
    ]
  }
}`

As you can see there are no bounding boxes, field names or values. How to get the annotated test data?

**Comment by sulc** [comment]
@Davo00 , to avoid research overfitting to the test set, the test set is indeed shared without annotations. Instead, you can evaluate your post-competition submissions at https://rrc.cvc.uab.es/?ch=26 .

**Comment by Davo00** [comment]
@sulc thanks for the quick response, I have registered and tried uploading the sample provided on the web page. This resulted into an error that the ground truth path is not found. Is there an issue or is it supposed to be like that?

![image](https://github.com/user-attachments/assets/2dd059de-aa0e-4a1a-b56b-34ba55753c95)
![image](https://github.com/user-attachments/assets/48f29d2d-25e5-44dd-9d83-b33415e3e413)

**Comment by simsa-st** [comment]
Hi @Davo00, thank you for the report, I confirm the submissions are currently broken. I have contacted the admins of the Robust Reading Competition portal to help me fix this.
